samcli/hook_packages/terraform/main.py

```diff
@@ -1,7 +1,9 @@
 """
-Module for Terraform hooks
+Module for Terraform hook entry points
 """
+from .hooks import TerraformHooks
 
 
-def prepare(self, options: dict) -> dict:
-    pass
+def prepare(params: dict) -> dict:
+    tf_hooks = TerraformHooks()
```

Comment:
should TerraformHooks be a module instead of a class? I feel like we don't use much functionality of classes (aside from creating a Hooks abstract class and implementing that)

Reply:
changed to module